dashboard-for-desktop/articles/dashboard-designer/providing-data/binding-to-federated-data-source.md

```diff
@@ -17,15 +17,39 @@ To create a federated data source with the Data Source Wizard, perform the follo
 
    ![Choose_federated_data_source](../../../images/choose-federated-data-source.png)
 
-3. The [Query Builder](../../dashboard-designer/working-with-data/using-the-query-builder.md) dialog appears that displays the available data sources.
+3. The [Query Builder](../../dashboard-designer/working-with-data/using-the-query-builder.md) dialog displays the available data sources.
 
    ![Query_Builder_available_sources](../../../images/query-builder-available-sources.png)
 
-4. Drag-and-drop the data sources, specify the joins and check the columns to include them in the query. When you are finished, click **OK**.
+4. Select the **Query Type**:
+
+   **Join**
+
+   Combines rows from two or more tables based on a column they share. The join type specifies records that have matching values in both tables.
+     
+    Drag-and-drop the data sources, specify the related column to create the relationship between tables, and select the columns you want to include in the query. Columns included in the query are displayed in the bottom pane where you can configure their settings.
 
    ![Federated_data_source_settings](../../../images/federated-source-settings.png)
 
-5. The [Data Source Browser](../ui-elements/data-source-browser.md) displays the newly created Federation Data Source:
+   **Union**
+
+   Combines rows from two or more tables into a single data set and removes duplicate rows in merged tables. You can only create a union query for data sources that contain similar columns with the same names. Data types of such columns should be [implicitly converted](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions#implicit-conversions).
+
+   Drag-and-drop the data sources to combine them in one. Unlike the join type, you cannot specify which columns to include in the union query. Columns included in the query are displayed in the bottom pane where you can specify the aliases of the newly created columns.
```

Comment:
For my understanding, are multiple data sources being combined into a single data source?

Reply:
In general - yes, https://www.essentialsql.com/what-is-the-difference-between-a-join-and-a-union/

Reply:
```suggestion
   Drag-and-drop the data sources to combine them into a single data source. Unlike a join query, you cannot specify columns to include in a union query. Columns included in the query are displayed in the bottom pane, where you can specify the aliases of the newly created columns.
```